### PR TITLE
rename/remove HAL_BUZZER_ON to DEFAULT_BUZZ_ON_LVL

### PR DESCRIPTION
**Pull request by Hwurzburg**
removes HAL_BUZZER_ON from all hwdef to save a bit of space and renames it to new pattern

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/scripts/convert_betaflight_unified.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def convert_file(fname, board_id):
     f.write('''\n# Beeper
 %s BUZZER OUTPUT GPIO(80) LOW
 define HAL_BUZZER_PIN 80
-define HAL_BUZZER_ON 1
+define DEFAULT_BUZZ_ON_LVL 1
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/scripts/convert_betaflight_unified.py:242]: I think we want to remove this - I'd bet a lot of the ones you're removing came in this way.

**Review by peterbarker — CHANGES_REQUESTED** [review]
Missed a few:

```
pbarker@fx:~/rc/ardupilot((HEAD detached at Hwurzburg/HAL_BUZZER_ON))$ git grep HAL_BUZZER_ON
libraries/AP_HAL_ChibiOS/hwdef/VRBrain-v51/hwdef.dat:define HAL_BUZZER_ON  1
libraries/AP_HAL_ChibiOS/hwdef/VRBrain-v52/hwdef.dat:define HAL_BUZZER_ON  1
libraries/AP_HAL_ChibiOS/hwdef/VRBrain-v54/hwdef.dat:define HAL_BUZZER_ON  1
libraries/AP_HAL_ChibiOS/hwdef/VRCore-v10/hwdef.dat:define HAL_BUZZER_ON  1
libraries/AP_HAL_ChibiOS/hwdef/VRUBrain-v51/hwdef.dat:define HAL_BUZZER_ON  1
```

File: libraries/AP_HAL_ChibiOS/hwdef/scripts/convert_betaflight_unified.py (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/scripts/convert_betaflight_unified.py]: yep

**Comment by Hwurzburg** [comment]
the double space before the "1" eluded my global search and replace....thanks

**Review by peterbarker — APPROVED** [review]

**Pull request merged**